binaries/README.md

```diff
@@ -73,3 +73,20 @@
     
      ##### Windows:
      Conda install is currently not supported. Please use pip install command instead.
+
+# Uploading packages for staging
+1. Export the following environment variables for TestPypi and anaconda.org authentication
+   ```
+   export CONDA_TOKEN=<>
+   export TWINE_USERNAME=<>
+   export TWINE_PASSWORD=<>
+   ```
+2. Edit `upload.py` to change the CONDA_USER if necessary
+3. Run the following commands to build the packages, and then upload them to staging repos
+   ```
+   python3 binaries/conda/build_packages.py --install-conda-dependencies
+   exec bash
```

Comment:
To my understanding, these steps should be covered by func  install_miniconda() and  func  install_conda_build(). They should be removed at here.